makefile

```diff
@@ -0,0 +1,15 @@
+QKCDIR = `pwd`/qkchash
+GRPCDIR = `pwd`/quarkchain/cluster
+MAKE = make
+
+all: generate subsystem
+
+generate:
+	python -m grpc_tools.protoc --proto_path=$(GRPCDIR) --python_out=$(GRPCDIR) --grpc_python_out=$(GRPCDIR) grpc_client.proto
+
+subsystem:
+	cd $(QKCDIR) && $(MAKE)
+
+clean:
+	rm -f $(QKCDIR)/qkchash libqkchash.so qkchash_llrb
```

Comment:
seems `make -C` can help you manage multiple Makefiles

Reply:
Done.